src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -30904,56 +31417,6 @@ void symbolValidator(ModuleSymbol m)
             }
         }
 
-        [Fact, WorkItem(35949, "https://github.com/dotnet/roslyn/issues/35949")]
-        public void NotNull_Complexity()
```

Comment:
Why get rid of this?

Reply:
Oh, it was moved. Consider not moving it.

---
In reply to: [303566828](https://github.com/dotnet/roslyn/pull/37201#discussion_r303566828) [](ancestors = 303566828)

Reply:
I moved it because it takes 1 minute to run ;-)

---
In reply to: [303566936](https://github.com/dotnet/roslyn/pull/37201#discussion_r303566936) [](ancestors = 303566936,303566828)